numpy/distutils/intelccompiler.py

```diff
@@ -16,10 +17,19 @@ class IntelCCompiler(UnixCCompiler):
     def __init__(self, verbose=0, dry_run=0, force=0):
         UnixCCompiler.__init__(self, verbose, dry_run, force)
 
+        if platform.system() == "Windows":
+            self.executables['version_cmd'] = ["icl" , "-dummy"]
+            self.version_pattern = r'Version\s*(?"version"[\d.]+)'
+        else:
+            self.executables['version_cmd'] = ["icc", "-dumpversion"]
+            self.version_pattern = r'^(?"version"[\d.]$)'
+
+        # Using -qopenmp leads to overlinking if C code does not use OpenMP features
+        # like in the case of NumPy. If an extension requires -qopenmp, it should be
+        # explicitly set for that extension
         v = self.get_version()
-        mpopt = 'openmp' if v and v < '15' else 'qopenmp'
-        self.cc_exe = ('icc -fPIC -fp-model strict -O3 '
-                       '-fomit-frame-pointer -{}').format(mpopt)
+        mpopt = '-openmp' if v and v < '15' else '-qopenmp'
+        self.cc_exe = 'icc -fPIC -fp-model strict -O3 -fomit-frame-pointer ' # + mpopt
```

Comment:
```suggestion
        self.cc_exe = 'icc -fPIC -fp-model strict -O3 -fomit-frame-pointer '
```
If the code is being removed, lets remove it completely.